portal-api/src/v1/email/index.js

```diff
@@ -1,15 +1,13 @@
-const { NotifyClient } = require('notifications-node-client');
-require('dotenv').config();
-
-const notifyClient = new NotifyClient(process.env.GOV_NOTIFY_API_KEY);
+const externalApi = require('../../external-api/api');
 
 const sendEmail = async (templateId, sendToEmailAddress, emailVariables) => {
-  const personalisation = emailVariables;
```

Comment:
Can we please ad `templateId`, `sendToEmailAddress` void validation beforing invoking the API please.